cmd/cmd_test.go

```diff
@@ -925,7 +924,9 @@ func (r *TestRepoRoot) CreateCommandRunner(ctx context.Context) *cobra.Command {
 	r.streams = ioes.NewIOStreams(in, out, out)
 	setNoColor(true)
 
-	return NewQriCommand(ctx, r.pathFactory, r.testCrypto, r.streams)
+	cmd := NewQriCommand(ctx, r.pathFactory, r.testCrypto, r.streams)
+	cmd.SetOutput(out)
```

Comment:
This fixes most of the problems with writing to stdout, but we should also change cmd/export.go to write to o.Out instead of fmt.Fprintf on [line 111](https://github.com/qri-io/qri/blob/master/cmd/export.go#L111). For some reason that line still prints out if I run the tests with "-v". Also, this commit is unrelated to the rest of this PR, could we move to a different PR?

Reply:
agreed, I'll submit a separate PR for this